### ISSUE TITLE
HPS for non-scientific ARE with RoBERTa

### ISSUE THREAD

**Issue by ArneBinder** [ArneBinder/pie-document-level#431]
references:
 - https://github.com/ArneBinder/pie-document-level/pull/410
 - https://github.com/ArneBinder/pie-document-level/pull/381

requires:
- #432

TODO:
- [x] aae2
- [x] argmicro
- [x] cdcp

<details><summary>click to see template</summary>

````
# DDD
- train command:
   - `screen -r pie-document-level-XXX`
```
$HOME/projects/pegasus-bridle/wrapper.sh python src/train.py \
hparams_search=are_argument_marked_text_classification \
hydra.sweeper.sampler.seed=12345 \
experiment=DDD_are \
transformer_model=FacebookAI/roberta-base \
datamodule.batch_size=128 \
datamodule.num_workers=12 \
trainer=gpu \
test=false \
name=dataset-DDD/task-are/hparams_search/roberta \
--multirun
```
- job: XXX
- status: TODO
````

</details>

## result at 1000k cumulative steps (per dataset):

|                                                                | dataset-aae2-task-are-hparams_search-roberta-training | dataset-argmicro-task-are-hparams_search-roberta-training | dataset-cdcp-task-are-hparams_search-roberta-training |
| :------------------------------------------------------------- | :---------------------------------------------------- | :-------------------------------------------------------- | :---------------------------------------------------- |
| ('statistics', 'steps', 'total')                               | 1269413.0                                             | 1203896.0                                                 | 1279788.0                                             |
| ('statistics', 'runs', 'total')                                | 66                                                    | 1375                                                      | 83                                                    |
| ('statistics', 'runs', 'considered@t=1000000')                 | 53                                                    | 1133                                                      | 62                                                    |
| ('statistics', 'runtime', 'total')                             | 3 days 17:55:47                                       | 5 days 11:30:51                                           | 3 days 14:39:11                                       |
| ('statistics', 'runtime', 'considered@t=1000000')              | 2 days 22:51:37                                       | 4 days 10:41:06                                           | 2 days 18:58:25                                       |
| ('best_run', 'metrics', 'metric/micro/f1_without_tn/val')      | 0.5953277945518494                                    | 0.7358490824699402                                        | 0.38235294818878174                                   |
| ('best_run', 'config', 'callbacks.model_checkpoint.save_last') | False                                                 | False                                                     | False                                                 |
| ('best_run', 'config', 'datamodule.batch_size')                | 64                                                    | 128                                                       | 64                                                    |
| ('best_run', 'config', 'dataset_split')                        | test                                                  | test                                                      | test                                                  |
| ('best_run', 'config', 'model.learning_rate')                  | 3e-05                                                 | 1e-05                                                     | 1e-05                                                 |
| ('best_run', 'config', 'model.task_learning_rate')             | 0.0003                                                | 0.001                                                     | 0.001                                                 |
| ('best_run', 'config', 'predict')                              | False                                                 | False                                                     | False                                                 |
| ('best_run', 'config', 'serializer._target_')                  | src.serializer.JsonSerializer2                        | src.serializer.JsonSerializer2                            | src.serializer.JsonSerializer2                        |
| ('best_run', 'config', 'serializer.split')                     | test                                                  | test                                                      | test                                                  |
| ('best_run', 'config', 'taskmodule.max_argument_distance')     | 400                                                   | 50                                                        | 150                                                   |
| ('best_run', 'config', 'trainer.deterministic')                | False                                                 | False                                                     | False                                                 |
| ('best_run', 'config', 'trainer.max_epochs')                   | 150                                                   | 200                                                       | 200                                                   |
| ('best_run', 'config', 'trainer.min_epochs')                   | 150                                                   | 200                                                       | 200                                                   |
| ('best_run', 'statistics', 'steps')                            | 66600.0                                               | 1200.0                                                    | 37800.0                                               |
| ('best_run', 'statistics', 'cumulative_steps')                 | 939655.0                                              | 394710.0                                                  | 943400.0                                              |
| ('best_run', 'statistics', 'runtime')                          | 0 days 04:38:41                                       | 0 days 00:09:50                                           | 0 days 02:32:24                                       |
| ('best_run', 'statistics', 'cumulative_runtime')               | 2 days 22:28:32                                       | 1 days 15:18:43                                           | 2 days 17:41:35                                       |
| ('best_run', 'other', 'name')                                  | dauntless-serenity-46                                 | fresh-plasma-435                                          | solar-dragon-59                                       |

**Comment by ArneBinder** [comment]
# aae2
- wandb project: https://wandb.ai/arne/dataset-aae2-task-are-hparams_search-roberta-training
- train command:
   - `screen -r pie-document-level-2`
```
$HOME/projects/pegasus-bridle/wrapper.sh python src/train.py \
hparams_search=are_argument_marked_text_classification \
hydra.sweeper.sampler.seed=12345 \
experiment=aae2_are \
transformer_model=FacebookAI/roberta-base \
datamodule.batch_size=64 \
datamodule.num_workers=12 \
trainer=gpu \
test=false \
name=dataset-aae2/task-are/hparams_search/roberta \
--multirun
```
- job: 1723002
- status: CANCELED

## start another job
- partition: `H100-Trails`
- train command:
   - `screen -r pie-document-level-5`
```
$HOME/projects/pegasus-bridle/wrapper.sh python src/train.py \
hparams_search=are_argument_marked_text_classification \
hydra.sweeper.sampler.seed=1234 \
experiment=aae2_are \
transformer_model=FacebookAI/roberta-base \
datamodule.batch_size=64 \
datamodule.num_workers=12 \
trainer=gpu \
test=false \
name=dataset-aae2/task-are/hparams_search/roberta \
--multirun
```
- job: 1724039
- status: CANCELED

**Comment by ArneBinder** [comment]
# argmicro
- wandb project: https://wandb.ai/arne/dataset-argmicro-task-are-hparams_search-roberta-training
- train command:
   - `screen -r pie-document-level-3`
```
$HOME/projects/pegasus-bridle/wrapper.sh python src/train.py \
hparams_search=are_argument_marked_text_classification \
hydra.sweeper.sampler.seed=12345 \
experiment=argmicro_are \
transformer_model=FacebookAI/roberta-base \
datamodule.batch_size=128 \
datamodule.num_workers=12 \
trainer=gpu \
test=false \
name=dataset-argmicro/task-are/hparams_search/roberta \
--multirun
```
- job: 1722582
- status: TERMINATED (time limit)

## restart
 - `screen -r pie-document-level-8`
```
$HOME/projects/pegasus-bridle/wrapper.sh python src/train.py \
hparams_search=are_argument_marked_text_classification \
hydra.sweeper.sampler.seed=1234 \
experiment=argmicro_are \
transformer_model=FacebookAI/roberta-base \
datamodule.batch_size=128 \
datamodule.num_workers=12 \
trainer=gpu \
test=false \
name=dataset-argmicro/task-are/hparams_search/roberta \
--multirun
```
- job: 1727570
- status: CANCELED (enough total steps)

**Comment by ArneBinder** [comment]
# cdcp
- wandb project: https://wandb.ai/arne/dataset-cdcp-task-are-hparams_search-roberta-training
- train command:
   - `screen -r pie-document-level-4`
```
$HOME/projects/pegasus-bridle/wrapper.sh python src/train.py \
hparams_search=are_argument_marked_text_classification \
hydra.sweeper.sampler.seed=12345 \
experiment=cdcp_are \
transformer_model=FacebookAI/roberta-base \
datamodule.batch_size=64 \
datamodule.num_workers=12 \
trainer=gpu \
test=false \
name=dataset-cdcp/task-are/hparams_search/roberta \
--multirun
```
- job: 1723004
- status: CANCELED

## start another job
- train command:
   - `PARTITION=H100-Trails`
   - `screen -r pie-document-level-6`
```
$HOME/projects/pegasus-bridle/wrapper.sh python src/train.py \
hparams_search=are_argument_marked_text_classification \
hydra.sweeper.sampler.seed=1234 \
experiment=cdcp_are \
transformer_model=FacebookAI/roberta-base \
datamodule.batch_size=64 \
datamodule.num_workers=12 \
trainer=gpu \
test=false \
name=dataset-cdcp/task-are/hparams_search/roberta \
--multirun
```
- job: 1724040
- status: CANCELED
